src/app/[locale]/_component/header/header.tsx

```diff
@@ -23,25 +23,26 @@ const Header = (): React.ReactElement => {
     },
   ];
 
-  // eslint-disable-next-line react-hooks/exhaustive-deps
-  const userMenuActions =
-    isLoggedIn && userLogged?.spidLevel !== JWT_SPID_LEVEL_L1
-      ? [
-          ...userMenuActionsBasic,
-          {
-            id: '1',
-            label: 'Vai al profilo',
-            onClick: () => {
-              pushWithLocale(ROUTES.PROFILE);
+  const userMenuActions = useMemo(
+    () =>
+      isLoggedIn && userLogged?.spidLevel !== JWT_SPID_LEVEL_L1
+        ? [
+            ...userMenuActionsBasic,
+            {
+              id: '1',
+              label: 'Vai al profilo',
+              onClick: () => {
+                pushWithLocale(ROUTES.PROFILE);
+              },
+              icon: <ManageAccountsIcon fontSize="small" color="inherit" />,
             },
-            icon: <ManageAccountsIcon fontSize="small" color="inherit" />,
-          },
-        ]
-      : userMenuActionsBasic;
+          ]
+        : userMenuActionsBasic,
+    [isLoggedIn, userLogged]
+  );
```

Comment:
Well done! This fix an additional re-render! I suggest to provide the full list of dependencies the lint wants, even if someone is not necessary, rather then silencing the warning with a lint directive. Making an exhausive list allow us to see again the warning if we add another dependency to the hook in the future... I know that it seems ugly and unnecessary now.